ui/components/TracePage/TracePage.tsx

```diff
@@ -454,10 +452,21 @@ const CallChildren = ({
 const isObjectLike = (value: unknown): value is object =>
   value !== null && typeof value === "object";
 
-const getFirstDescendant = (value: unknown): unknown =>
-  isObjectLike(value) ? getFirstDescendant(Object.values(value)[0]) : value;
+const isStringArray = (value: unknown): value is unknown[] =>
+  Array.isArray(value) && typeof value[0] === 'string';
 
-const getString = (value: any): string => {
+const getFirstDescendant = (value: unknown): unknown => {
+  if (isObjectLike(value) && !isStringArray(value)) {
+    return getFirstDescendant(Object.values(value)[0])
+  } else if (isStringArray(value)) {
+    return value.filter(el => typeof el === 'string');
+  } else {
+    return value;
+  }
+}
+
+const getStrings = (value: any): string[] => {
```

Comment:
Can you use unknown instead of any?

Reply:
was following the original code, tried changing to unknown and it breaks type checking for the isObjectLike below for some reason - seems like typescript can't infer the types from the nested ifs.

Reply:
Okay, seems fine to leave for now, I'll take a look later.